library/pem.c

```diff
@@ -230,15 +355,13 @@ int mbedtls_pem_read_buffer( mbedtls_pem_context *ctx, const char *header, const
     size_t len;
     unsigned char *buf;
     const unsigned char *s1, *s2, *end;
-#if defined(MBEDTLS_MD5_C) && defined(MBEDTLS_CIPHER_MODE_CBC) &&         \
-    ( defined(MBEDTLS_DES_C) || defined(MBEDTLS_AES_C) )
+#if defined(PEM_RFC1421 )
```

Comment:
```suggestion
#if defined( PEM_RFC1421 )
```

Reply:
Actually, no. We put spaces inside parentheses _except_ for `defined` and casts. Don't ask me why.

Reply:
Right, the other way around :D